lnst/Controller/ContainerPoolManager.py

```diff
@@ -207,7 +213,15 @@ def _create_container(self, name: str, reqs: dict):
 
         return container, machine
 
-    def _create_network(self, network_name: str):
+    def _create_network(self, network_name: str, plugin: str="netavark"):
```

Comment:
you probably don't need the default value in this private method since you already define the default in `__init__`